src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt

```diff
@@ -210,6 +214,54 @@ class AdminController(
     return "/admin/plantingSite"
   }
 
+  @GetMapping(
+      "/plantingSite/{plantingSiteId}/downloadZone/{plantingZoneId}",
+      produces = [MediaType.APPLICATION_JSON_VALUE])
+  @ResponseBody
+  fun downloadPlantingZone(
+      @PathVariable plantingSiteId: PlantingSiteId,
+      @PathVariable plantingZoneId: PlantingZoneId,
+      response: HttpServletResponse
+  ): Map<String, Any> {
+    response.addHeader(
+        HttpHeaders.CONTENT_DISPOSITION, """attachment; filename="zone-${plantingZoneId}.json"""")
+
+    val zone =
+        plantingSiteStore.fetchSiteById(plantingSiteId).plantingZones.first {
+          it.id == plantingZoneId
+        }
+
+    // Return a GeoJSON document that can be viewed using tools such as geojson.io.
+    return mapOf(
+        "type" to "FeatureCollection",
+        "features" to
+            listOf(
+                mapOf(
+                    "type" to "Feature",
+                    "properties" to emptyMap<String, Any>(),
```

Comment:
In case it helps with debug use-cases, we can add the name of the zones and subzones. Of course we can't do that for monitoring plots, so maybe this partial suggestion is not so useful.